### ISSUE TITLE
feat: consume license key from args even outside EASY API

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2482]
Today a license buyer was using a project that depended on this library. They were unable to tell exactly where to add the license due to all of the code abstraction. To prevent this issue, make sure to check process args for `--license-key` or  `-l` just in case.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.30.11](https://github.com/open-wa/wa-automate-nodejs/compare/4.30.10...4.30.11) (2022-02-02)

- Bump chrome-launcher from 0.14.0 to 0.15.0 [`#2479`](https://github.com/open-wa/wa-automate-nodejs/pull/2479)
- Bump swagger-ui-dist from 4.1.3 to 4.4.1 [`#2480`](https://github.com/open-wa/wa-automate-nodejs/pull/2480)
- Bump @typescript-eslint/eslint-plugin from 5.8.0 to 5.10.2 [`#2477`](https://github.com/open-wa/wa-automate-nodejs/pull/2477)
- Bump @discordjs/collection from 0.2.1 to 0.5.0 [`#2478`](https://github.com/open-wa/wa-automate-nodejs/pull/2478)
- Bump winston from 3.3.3 to 3.5.1 [`#2475`](https://github.com/open-wa/wa-automate-nodejs/pull/2475)
- Bump @types/node from 17.0.13 to 17.0.14 [`#2474`](https://github.com/open-wa/wa-automate-nodejs/pull/2474)
- Bump command-line-args from 5.2.0 to 5.2.1 [`#2466`](https://github.com/open-wa/wa-automate-nodejs/pull/2466)
- Bump @types/changelog-parser from 2.7.1 to 2.8.1 [`#2473`](https://github.com/open-wa/wa-automate-nodejs/pull/2473)
- Bump winston-daily-rotate-file from 4.5.5 to 4.6.0 [`#2472`](https://github.com/open-wa/wa-automate-nodejs/pull/2472)
- Bump rxjs from 7.4.0 to 7.5.2 [`#2471`](https://github.com/open-wa/wa-automate-nodejs/pull/2471)
- Bump prettier from 2.4.1 to 2.5.1 [`#2470`](https://github.com/open-wa/wa-automate-nodejs/pull/2470)
- Bump typescript from 4.4.4 to 4.5.5 [`#2465`](https://github.com/open-wa/wa-automate-nodejs/pull/2465)
- Bump @typescript-eslint/parser from 5.10.1 to 5.10.2 [`#2469`](https://github.com/open-wa/wa-automate-nodejs/pull/2469)
- Bump ts-json-schema-generator from 0.95.0 to 0.97.0 [`#2468`](https://github.com/open-wa/wa-automate-nodejs/pull/2468)
- Bump release-it from 14.11.6 to 14.12.4 [`#2467`](https://github.com/open-wa/wa-automate-nodejs/pull/2467)
- Bump socket.io from 4.2.0 to 4.4.1 [`#2463`](https://github.com/open-wa/wa-automate-nodejs/pull/2463)
- Bump socket.io-client from 4.2.0 to 4.4.1 [`#2462`](https://github.com/open-wa/wa-automate-nodejs/pull/2462)
- Bump ava from 3.15.0 to 4.0.1 [`#2461`](https://github.com/open-wa/wa-automate-nodejs/pull/2461)
- Bump @types/marked from 3.0.1 to 4.0.2 [`#2460`](https://github.com/open-wa/wa-automate-nodejs/pull/2460)
- Bump eslint from 8.1.0 to 8.8.0 [`#2459`](https://github.com/open-wa/wa-automate-nodejs/pull/2459)
- Bump puppeteer from 13.0.0 to 13.1.3 [`#2458`](https://github.com/open-wa/wa-automate-nodejs/pull/2458)
- Bump marked from 3.0.8 to 4.0.12 [`#2446`](https://github.com/open-wa/wa-automate-nodejs/pull/2446)
- Bump @types/node from 16.11.0 to 17.0.13 [`#2456`](https://github.com/open-wa/wa-automate-nodejs/pull/2456)
- Bump @types/shelljs from 0.8.9 to 0.8.11 [`#2457`](https://github.com/open-wa/wa-automate-nodejs/pull/2457)
- Bump axios from 0.21.4 to 0.25.0 [`#2423`](https://github.com/open-wa/wa-automate-nodejs/pull/2423)
- Bump @typescript-eslint/parser from 4.33.0 to 5.10.1 [`#2430`](https://github.com/open-wa/wa-automate-nodejs/pull/2430)
- Bump qs from 6.10.1 to 6.10.3 [`#2414`](https://github.com/open-wa/wa-automate-nodejs/pull/2414)
- Bump shelljs from 0.8.4 to 0.8.5 [`#2420`](https://github.com/open-wa/wa-automate-nodejs/pull/2420)
- Release 4.30.11 [`0b4dc5c`](https://github.com/open-wa/wa-automate-nodejs/commit/0b4dc5c35eb95ea05747d3ab954bb147e5d2f13f)
- ⬇️ downgrade typescript dep to prevent doc gen failure. unblocks #2482 [`ae075a7`](https://github.com/open-wa/wa-automate-nodejs/commit/ae075a7f407b30febcccf2ddea3a88cd0c370e1b)
- 🐛 build:  fix marked is not a function, unblocks #2482 [`a80cf60`](https://github.com/open-wa/wa-automate-nodejs/commit/a80cf606ae9d144f1fc9fd0664ab2061dc089278)
- ✨ feat: license can be set via `-l`/`--license-key` even with custom code #2482 [`90c45f5`](https://github.com/open-wa/wa-automate-nodejs/commit/90c45f54cbe10abc603a009cd6e425b3d19f1686)
- updated types-only package [`85512fe`](https://github.com/open-wa/wa-automate-nodejs/commit/85512fe10536eaf9574a0678cb454ef34f2caf9d)
